thepracticaldev/dev.to
# Created the spinner for the image uploader

## What type of PR is this? (check all applicable)


## Description

Creates the spinner component that can be used anywhere, but for now it's for the image uploaded in the editor v2.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

*The skip is simply the animated gif ending*

![Jun-18-2020 18-41-47](https://user-images.githubusercontent.com/833231/85078864-6b7ec380-b193-11ea-9cc2-fd8b17bdf504.gif)

![image](https://user-images.githubusercontent.com/833231/85078921-8f420980-b193-11ea-9328-677f51a75666.png)

![image](https://user-images.githubusercontent.com/833231/85078934-95d08100-b193-11ea-9c27-00ab4987a69a.png)

![image](https://user-images.githubusercontent.com/833231/85078942-9bc66200-b193-11ea-8103-130d47fa10b0.png)

![image](https://user-images.githubusercontent.com/833231/85078992-b6004000-b193-11ea-8f23-10354e6dbb9c.png)

To see the spinner in action:

* Navigate to https://deploy-preview-8781--storybookdevto.netlify.app/?path=/story/4-app-components-shared-spinner--default
* Use the theme drop down to see it rendered for different themes.

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![A dolphin spinning in the water](https://media.giphy.com/media/AZcYQr9FMKypW/giphy.gif)
